docs/function/handlers.md

```diff
@@ -224,6 +224,22 @@ Note that path parameters (e.g. `{id}` in the example above) are available as re
 $id = $request->getAttribute('id');
 ```
 
+### Event and context
+
+The HTTP request event and context are available as attributes on the request:
```

Comment:
```suggestion
The API Gateway event and Lambda context are available as attributes on the request:
```

(because it's not the HTTP request, it's the low-level API Gateway event)